packages/IMAPClient-Core.package/ICConnectionHandler.class/instance/sendCommand..st

```diff
@@ -1,14 +1,5 @@
 connection
-sendCommand: aCommand
-	
-	| hash conOK |
-	
-	hash := self accountInfo hash.
-	conOK := true.
-	
-	self storedCommands add: aCommand for: self accountInfo hash.
-	
-	[conOK and: (self storedCommands isEmptyFor: hash) not] whileTrue: [
```

Comment:
We should start identifying accounts by their name or a dedicated identifier instead of some weird hash.